pxd.c

```diff
@@ -1024,13 +1024,14 @@ static void pxd_rq_fn(struct request_queue *q)
 		fproot = &req->fproot;
 		fp_root_context_init(fproot);
 
+#ifdef __PX_FASTPATH__
```

Comment:
introduce compile time options to remove fastpath support and add stubs as necessary.